process/project-lifecycle-documents/model_signing_sandbox_stage.md

```diff
@@ -0,0 +1,97 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+
+The project has 4 maintainers from 3 different organizations:
+
+*  Laurent Simon, Google, @laurentsimon
+*  Daniel Major, NVidia,
+*  Eoin Wickens, HiddenLayer, @EWickens
+*  Mihai Maruseac, Google, @mihaimaruseac
+
+### Mission of the project
+
+The project must be aligned with the OpenSSF mission and either be a novel
+approach for existing areas, address an unfulfilled need, or be initial code
+needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF
+projects collaborate with the existing project rather than seek a new project.
+
+> Create a cryptographic signing specification for artificial intelligence and
+> machine learning models, addressing challenges such as very large models that
+> can be used separately, and the signing of multiple disparate file formats
+> held within a directory. This specification may have wider applicability to
+> signing directories of multiple arbitrary file formats. This specification may
+> later be proposed as a formal standard.
+
+#### Specific Goals Include:
+
+* Develop standards for efficient hashing of large models
+* Develop standards for efficient verification of models that contain multiple
+  formats in the same place
+* Develop standards for efficient verification of models at inference time.
```

Comment:
Just to be clear, by verification here, we mean hash-based integrity checking? Verification is a pretty broad term, so if the scope for this project is narrower, I think we should clarify that in the goals.

Reply:
Thank you for the clarification! Updated to make it clear that both here and in line 29 we only mean integrity of the model bytes.